apps/frontend/src/mixins/EvaluationMixin.ts

```diff
@@ -1,14 +1,20 @@
+import {ServerModule} from '@/store/server';
 import {IGroup} from '@heimdall/interfaces';
 import {Component, Vue} from 'vue-property-decorator';
 import {IVuetifyItems} from '../utilities/helper_util';
 
 @Component({})
 export default class EvaluationMixin extends Vue {
   convertGroupsToIVuetifyItems(groups: IGroup[] | undefined): IVuetifyItems[] {
+    const currentUserId = ServerModule.userInfo.id;
     if (!groups) {
       return [];
     }
-    return groups.map((group) => {
+    const validatedGroups = groups.filter((group) =>
```

Comment:
I wasn't aware we were already tracking the groups that the user was in.  Unless I am thinking about something wrong, we wouldn't need the intersection since my logic was just to determine the groups the user was in anyways.

Reply:
For one of them we just want the groups the user is in.  But for the other we only want the groups that are applied to the evaluation that the user is also in.  Having the function return the set of all groups that the user is in would be incorrect in that case. 

https://github.com/search?q=repo%3Amitre%2Fheimdall2+convertgroupstoivuetifyitems&type=code

Reply:
Ah, I get what you mean. I just pushed up the fix now.